Improvement : Responsive data nilai

dika841 opened the pull request:
Slicing form login keuangan
![image](https://github.com/himatifdevteam/uninus/assets/95964808/adf0f2e0-60cb-4590-8b5e-186bacdca9e0)

![image](https://github.com/himatifdevteam/uninus/assets/95964808/e896d9f5-cdd6-4928-84ec-2f59d0a6f6ca)

Mobile :Horizontal Scroll 
![image](https://github.com/himatifdevteam/uninus/assets/95964808/1275c159-d62c-4c2e-8b74-c24253a1bdea)

dika841 committed:
Improvement : Responsive data nilai

maulanasdqn commented on libs/web/modules/src/dashboard/user/registrasi/biodata/section/data-nilai/index.tsx:
```diff
@@ -87,8 +87,8 @@ export const DataNilaiSection: FC = (): ReactElement => {
 
-  const labelSemesterStyle = "text-[10px] md:text-base";
-  const labelCourseStyle = "text-[8px] lg:text-base font-bold pr-4 lg:pr-0";
+  const labelSemesterStyle = "text-[13px] md:text-base";
```
jadikeun tailwind alias di tailwind config

https://stackoverflow.com/questions/73898178/aliases-for-tailwind-classes

maulanasdqn reviewed (changes requested):
cek

dependabot[bot] committed:
build(deps-dev): bump jest and @types/jest (#1017)

Bumps [jest](https://github.com/jestjs/jest/tree/HEAD/packages/jest) and [@types/jest](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/jest). These dependencies needed to be updated together.

Updates `jest` from 29.6.2 to 29.6.4
- [Release notes](https://github.com/jestjs/jest/releases)
- [Changelog](https://github.com/jestjs/jest/blob/main/CHANGELOG.md)
- [Commits](https://github.com/jestjs/jest/commits/v29.6.4/packages/jest)

Updates `@types/jest` from 29.5.3 to 29.5.4
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/jest)

---
updated-dependencies:
- dependency-name: jest
  dependency-type: direct:development
  update-type: version-update:semver-patch
- dependency-name: "@types/jest"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump cypress from 12.17.4 to 13.1.0 (#1018)

Bumps [cypress](https://github.com/cypress-io/cypress) from 12.17.4 to 13.1.0.
- [Release notes](https://github.com/cypress-io/cypress/releases)
- [Changelog](https://github.com/cypress-io/cypress/blob/develop/CHANGELOG.md)
- [Commits](https://github.com/cypress-io/cypress/compare/v12.17.4...v13.1.0)

---
updated-dependencies:
- dependency-name: cypress
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump @aws-sdk/lib-storage from 3.400.0 to 3.405.0 (#1019)

Bumps [@aws-sdk/lib-storage](https://github.com/aws/aws-sdk-js-v3/tree/HEAD/lib/lib-storage) from 3.400.0 to 3.405.0.
- [Release notes](https://github.com/aws/aws-sdk-js-v3/releases)
- [Changelog](https://github.com/aws/aws-sdk-js-v3/blob/main/lib/lib-storage/CHANGELOG.md)
- [Commits](https://github.com/aws/aws-sdk-js-v3/commits/v3.405.0/lib/lib-storage)

---
updated-dependencies:
- dependency-name: "@aws-sdk/lib-storage"
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump @nestjs/platform-express from 10.2.2 to 10.2.4 (#1020)

Bumps [@nestjs/platform-express](https://github.com/nestjs/nest/tree/HEAD/packages/platform-express) from 10.2.2 to 10.2.4.
- [Release notes](https://github.com/nestjs/nest/releases)
- [Commits](https://github.com/nestjs/nest/commits/v10.2.4/packages/platform-express)

---
updated-dependencies:
- dependency-name: "@nestjs/platform-express"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

maulanasdqn committed:
Update README.md

dika841 committed:
Improvement : Responsive data nilai

maulanasdqn reviewed (approved):
nah ini baru good
